snipssonos/services/deezer/music_search_and_play_service.py

```diff
@@ -0,0 +1,126 @@
+import requests
+import logging
+
+from snipssonos.services.node.query_builder import NodeQueryBuilder
+from snipssonos.services.music.search_service import MusicSearchService
+
+from snipssonos.exceptions import MusicSearchProviderConnectionError
+
+
+class DeezerMusicSearchService(MusicSearchService):
+    SERVICE_NAME = "deezer"
+    DUMMY_LIST = [""]
+
+    def __init__(self):
+        self.query_builder = None
+
+    def set_node_query_builder(self, device_name):
```

Comment:
Isn't the name of this method too specific ? why do we need to specify `node` query builder ?

Reply:
I foresee that when having to create the action-custom to retrieve the user data from Deezer we will have to use a different query builder which will hit the Deezer API instead of using the Node Sonos server. At that point, we will have a function called `set_deezer_api_query_builder` to set the right query builder.

Reply:
After offline discussion, I will be passing the query builder as a parameter in the function.